editor/src/ui-tools/quick-action/quick-action-menu-ui.ts

```diff
@@ -143,45 +143,61 @@ export class InfoQuickActionMenu extends SimpleMenu {
   createMenuBody(bodyDiv: HTMLElement): void {
     const menu = createElement('div', ['bar-menu-text']);
     bodyDiv.appendChild(menu);
-
-    this.action.markers.forEach(marker => menu.appendChild(this.createMarker(marker)));
-
     if (this.action.title) {
-      const title = createElement('p', ['simple-menu-header']);
-      title.textContent = this.action.title;
-      menu.appendChild(title);
+      menu.appendChild(this.createTitle(this.action.title));
     }
-
-    if (this.action.outerElement) {
-      const outerElement = createElement('p', ['simple-menu-text']);
-      outerElement.textContent = '[' + this.action.outerElement + ']';
-      menu.appendChild(outerElement);
-    }
-
+    this.action.markers.forEach(marker => menu.appendChild(this.createMarker(marker)));
     if (this.action.text) {
-      const converter = new Converter({ simpleLineBreaks: true });
-      const htmlText = converter.makeHtml(this.action.text);
-      const template = document.createElement('template');
-      template.innerHTML = htmlText;
-      const text = createElement('div', ['simple-menu-text']);
-      for (const child of template.content.childNodes) {
-        text.appendChild(child);
+      menu.appendChild(this.createDescription(this.action.text));
+    }
+    if (this.action.info) {
+      for (const [label, value] of Object.entries(this.action.info)) {
+        menu.appendChild(this.createInfo(label, value));
       }
-      menu.appendChild(text);
     }
+    menu.appendChild(this.createInfo('PID', this.action.elementId));
+  }
+
+  private createTitle(name: string): HTMLElement {
+    const title = createElement('p', ['simple-menu-header']);
+    title.textContent = name;
+    return title;
+  }
 
-    const elementId = createElement('p', ['simple-menu-text', 'simple-menu-small']);
-    elementId.textContent = `PID: ${this.action.elementId}`;
-    menu.appendChild(elementId);
+  private createInfo(infoLabel: string, infoValue: string): HTMLElement {
+    const info = createElement('p', ['simple-menu-text', 'simple-menu-small']);
+    const label = createElement('strong');
+    label.textContent = `${infoLabel}: `;
+    if (infoValue.includes('\n')) {
```

Comment:
May be better to have also something like a type hint?!?

Reply:
yes maybe... I don't know if its worth the effort, but maybe not so bad for the future.
you think something about:
`{'info': {'Code': {'typeInfo': 'MULTILINE', 'value': 'this is some code;'}}}`

instead of:
`{'info': {'Code': 'this is some code;'}}`

Reply:
currently the only difference is that if the value has a '\n' char then it is packed into a "pre" element. but yes maybe every code should be in a pre element. And I'm not sure if there are other values which can have a '\n'...

Reply:
{'info': {'Code': {'type': 'CODE', 'value': 'this is some code;'}}}

Reply:
> I will merge this and try this our an may create a new PR.

Perfect, go for it :+1: